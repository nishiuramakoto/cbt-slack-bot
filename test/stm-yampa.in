test
test2
:3
I'm a dumb

Labeling

No one is dumb

:5
At work I had difficulty concentrating
on coding.

I can do anything helpful to others

Lonely, Sad

Overgeneralization

I have done many things
helpful to others
