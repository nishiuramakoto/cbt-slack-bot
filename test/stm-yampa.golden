BotStart
BotWarn "unknown command:test"
BotWarn "unknown command:test2"
BotMethodStart CbtTripleColumnTechnique
BotStackPush   CbtTripleColumnTechnique CbtStackAT "I'm a dumb"
BotStackFreeze CbtTripleColumnTechnique CbtStackAT
BotStackPush   CbtTripleColumnTechnique CbtStackCD "Labeling"
BotStackFreeze CbtTripleColumnTechnique CbtStackCD
BotStackPush   CbtTripleColumnTechnique CbtStackRR "No one is dumb"
BotStackFreeze CbtTripleColumnTechnique CbtStackRR
BotMethodFinish CbtTripleColumnTechnique
BotMethodStart CbtFiveColumnTechnique
BotStackPush   CbtFiveColumnTechnique CbtStackST "At work I had difficulty concentrating"
BotStackPush   CbtFiveColumnTechnique CbtStackST "on coding."
BotStackFreeze CbtFiveColumnTechnique CbtStackST
BotStackPush   CbtFiveColumnTechnique CbtStackAT "I can do anything helpful to others"
BotStackFreeze CbtFiveColumnTechnique CbtStackAT
BotStackPush   CbtFiveColumnTechnique CbtStackEM "Lonely, Sad"
BotStackFreeze CbtFiveColumnTechnique CbtStackEM
BotStackPush   CbtFiveColumnTechnique CbtStackCD "Overgeneralization"
BotStackFreeze CbtFiveColumnTechnique CbtStackCD
BotStackPush   CbtFiveColumnTechnique CbtStackRR "I have done many things"
BotStackPush   CbtFiveColumnTechnique CbtStackRR "helpful to others"
BotStackFreeze CbtFiveColumnTechnique CbtStackRR
BotMethodFinish CbtFiveColumnTechnique
BotFinish
